webots_ros2_crazyflie/setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import find_packages, setup
+
+package_name = 'webots_ros2_crazyflie'
+data_files = []
+data_files.append(('share/ament_index/resource_index/packages', ['resource/' + package_name]))
+data_files.append(('share/' + package_name + '/launch', ['launch/robot_launch.py']))
+data_files.append(('share/' + package_name + '/worlds', [
+    'worlds/crazyflie_apartment.wbt', 'worlds/.crazyflie_apartment.wbproj',
+]))
+data_files.append(('share/' + package_name + '/resource', [
+    'resource/crazyflie_webots.urdf'
+]))
+data_files.append(('share/' + package_name, ['package.xml']))
+
+setup(
+    name=package_name,
+    version='0.0.0',
```

Comment:
Version 0.0.0 looks odd...

Reply:
ah yes ofcourse

I've updated it to be similar to the other packages which say 2023.1.1, so I'm assuming that is the date of the update? now for this package it says 2024.1.3 (not sure if that is the intention).

Reply:
No, it actually refers to the version of Webots. I don't recall the exact logic, but I believe that R2023b translates to 2023.1.1, so you should put this value instead.

Reply:
Alright will get right on it